packages/a11y-base/src/focus-trap-controller.js

```diff
@@ -87,7 +87,7 @@ export class FocusTrapController {
     instances.push(this);
 
     if (this.__focusedElementIndex === -1) {
-      this.__focusableElements[0].focus();
+      this.__focusableElements[0].focus({ focusVisible: true });
```

Comment:
IMO this should replicate what the native `:focus-visible` behavior is, which would be to only show the focus when using the keyboard.

Reply:
Thanks, sounds reasonable. Changed.